cmake/Modules/Findblis.cmake

```diff
@@ -0,0 +1,42 @@
+SET(BLIS_INCLUDE_SEARCH_PATHS
+       /usr/local/include/blis
+)
+
+SET(BLIS_LIB_SEARCH_PATHS
+       /usr/local/lib
+)
+
+FIND_PATH(BLIS_INCLUDE_DIR NAMES cblas.h blis.h PATHS ${BLIS_INCLUDE_SEARCH_PATHS})
+FIND_LIBRARY(BLIS_LIB NAMES libblis.so PATHS ${BLIS_LIB_SEARCH_PATHS})
```

Comment:
I still do not think this is correct.

See https://cmake.org/cmake/help/v3.0/command/find_library.html
and compare to, e.g., https://github.com/ROCmSoftwarePlatform/pytorch/blob/fcda9d1f1d2ff80865936aad7a064cebfe48d4fa/cmake/Modules/FindOpenBLAS.cmake#L34

this should read in my opinion
```
FIND_LIBRARY(BLIS_LIB NAMES blis PATHS ${BLIS_LIB_SEARCH_PATHS})
```

no?

Reply:
Thanks for the suggestion. We have corrected and updated the file. Kindly review.

Reply:
You changed it to `libblis`, not `blis`.

Reply:
Sorry for mistake! Corrected it now.